openbas-model/src/main/java/io/openbas/database/model/InjectTestStatus.java

```diff
@@ -0,0 +1,78 @@
+package io.openbas.database.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.openbas.database.converter.InjectStatusExecutionConverter;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.time.Instant;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Setter
+@Getter
+@Entity
+@Table(name = "injects_tests_statuses")
+public class InjectTestStatus extends BaseInjectStatus implements Base {
+
+  @JsonProperty("inject_title")
+  public String getInjectTitle() {
+    return inject.getTitle();
+  }
+
+  @JsonProperty("injector_contract")
+  public Optional<InjectorContract> getInjectContract() {
+    return inject.getInjectorContract();
+  }
+
+  @JsonProperty("inject_type")
+  private String getType() {
+    return inject.getInjectorContract()
+        .map(InjectorContract::getInjector)
+        .map(Injector::getType)
+        .orElse(null);
+  }
+
+  @CreationTimestamp
+  @Column(name = "status_created_at")
+  @JsonProperty("inject_test_status_created_at")
+  private Instant testCreationDate;
+
+  @UpdateTimestamp
+  @Column(name = "status_updated_at")
+  @JsonProperty("inject_test_status_updated_at")
+  private Instant testUpdateDate;
+
+  public static InjectTestStatus fromExecutionTest(Execution execution) {
+    InjectTestStatus injectTestStatus = new InjectTestStatus();
```

Comment:
You could include this method on you abstract class, and here you can make:  `public static InjectTestStatus fromExecutionTest(Execution execution) {
        InjectTestStatus injectTestStatus = new InjectTestStatus();
        injectTestStatus.**updateStatusFromExecution**(execution);
        return injectTestStatus;
    }`